server/sonar-web/src/main/js/apps/issues/sidebar/CreationDateFacet.js

```diff
@@ -22,13 +22,13 @@ import React from 'react';
 import { max } from 'lodash';
 import { intlShape } from 'react-intl';
 import DateFromNow from '../../../components/intl/DateFromNow';
-import { longFormatterOption } from '../../../components/intl/DateFormatter';
+import { formatterOption, longFormatterOption } from '../../../components/intl/DateFormatter';
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import of 'formatterOption'. ([javascript:S1128](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS1128))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3157&issues=AWIqkLZunhKRZuqhdv-6&open=AWIqkLZunhKRZuqhdv-6)

[ISSUE_KEY]: AWIqkLZunhKRZuqhdv-6